askbot/media/style/style.css

```diff
@@ -143,6 +205,37 @@ textarea {
   padding: 3px 5px;
   width: 398px;
 }
+/* changed by mohini */
```

Comment:
We don't add comments like this. That's what git history is for.

Reply:
You can add meaningful comments
Eg: this selector applies for the Contributors section.